src/settings/cli.rs

```diff
@@ -35,7 +35,7 @@ pub struct General {
     #[structopt(
         long,
         short = "n",
-        default_value = "1",
+        default_value = "4",
         env = "SERVER_THREADS_MULTIPLIER"
```

Comment:
I'm wondering why now we should default to `4` workers instead of `1` per core (default behavior).

Reply:
As this a multiplier of the number server cores anyway a factor of one should give the best CPU efficiency (more requests for the same compute) however it will not give the best latency efficiency which will benefit with more threads per core for improved concurrency. Its a balance between the two thus it depends on what you want to tune for, maybe 2 is a good compromise? Only testing will tell

Reply:
I understand. In general, 1 per core was just a ground default. But probably we could go with something that you suggested in your [other comment](https://github.com/static-web-server/static-web-server/pull/181#discussion_r1125742586) below.